src/main/java/com/google/devtools/build/lib/bazel/rules/python/python_stub_template.txt

```diff
@@ -314,6 +314,10 @@ def Main():
   if runfiles_envkey:
     new_env[runfiles_envkey] = runfiles_envvalue
 
+  # Don't prepend a potentially unsafe path to sys.path
```

Comment:
fwiw, i'd recommend also linking to the rules_python issue. The unsafe behavior this is trying to prevent isn't immediately obvious.
(apologies for the drive-by comment)

Reply:
👍 Good idea.